docs/config-file-package-configuration.yml.md

```diff
@@ -0,0 +1,99 @@
+# The package configuration file (Experimental)
+
+A package configuration sets up path excludes and license finding curations specific to a package (dependency) and 
+provenance. It is thus a similar concept to the `.ort.yml` file which allows to do similar configuration for projects as
+opposed to packages.
+
+## Specifying the target package and provenance
+
+Each package configuration applies exactly to one package-Id and provenance which must be specified. The provenance
+can either specify a `source artifact` or a `VCS location and revision`. Thus in case one scans the `source artifact`
+and the `VCS` for each package, then two package configurations would be required in order to address issues with both
+scans. For example configurations for `ansi-styles` 4.2.1 would specify the following... 
+  
+* to configure the VCS scan:
+```yaml
+  id: "NPM::ansi-styles:4.2.1"
+  vcs:
+    type: "Git"
+    url: "https://github.com/chalk/ansi-styles.git"
+    revision: "74d421cf32342ac6ec7b507bd903a9e1105f74d7"
+```
+* to configure the source artifact scan:
+```yaml
+  id: "Maven:org.ossreviewtoolkit.ort:1.2.3"
+  source_artifact_url: "https://registry.npmjs.org/ansi-styles/-/ansi-styles-3.2.1.tgz"
+```
+
+## Specifying path excludes and license finding curations
+
+The entries for path excludes and license finding curations have the same syntax and sematics as the analog entries for
+the `ort.yml`, see [Excluding paths](config-file-ort-yml.md#excluding-paths) and
+[Curating license findings](config-file-ort-yml.md#curating-license-findings).
```

Comment:
Even though it's probably explained behind the links, I think it would be helpful to start this section with a short explanation of what path excludes and license finding curations are used for. For example:

Path excludes are used to tell ORT that license findings in certain files can be ignored, for example because they belong to the documentation of the package. License finding curations are used to fix wrong scan results, for example if a wrong license was detected, or if a finding is a false positive.

Reply:
I've used your suggestion `as-is`.